aiounifi/interfaces/connectivity.py

```diff
@@ -71,6 +71,12 @@ async def login(self) -> None:
             "rememberMe": True,
         }
 
+        if hasattr(self.config, "site") and self.config.site is not None:
```

Comment:
site will always exist as part of the configuration class and it can only be a string so the if-statement is unnecessary.